### ISSUE TITLE
Unhelpful default colors

### ISSUE THREAD

**Issue by aclementev** [junegunn/rainbow_parentheses.vim#26]
Hi!

I was trying today the plugin, and found that for some reason, the default colors picked suck for the
`base16-atelier-dune` colorscheme. Most of them look basically the same as the default text. 

Here a capture of `:RainbowParenthesesColors` output:

![image](https://user-images.githubusercontent.com/17129652/77215079-8ac57080-6b12-11ea-9b05-f85e044f3b76.png)

Also, I am using terminal vim on Windows Terminal, with a base16 colorscheme with:
```vim
set background=dark
let base16colorspace=256
colorscheme base16-atelier-dune  " Same issue with base16-default-dark
```

**Comment by laoshaw** [comment]
that's true, it picks some colors that just disappear the parenthesis (invisible), better let user to pick a set of colors based on their own color scheme?
